Core/Enemy.py

```diff
@@ -0,0 +1,41 @@
+import arcade
+import math
+
+from Constants.Game import SPRITE_SIZE, SPRITE_SCALING_PLAYER
+from Constants.Physics import PLAYER_MOVEMENT_SPEED
+
+class Enemy(arcade.Sprite):
+
+    def __init__(self, x, y, path, walls):
+        self.x = x
+        self.y = y
+        self.speed = PLAYER_MOVEMENT_SPEED
+        self.path = path
+        self.obstacles = walls.layers[0].layer_data
+
+        # Create enemy sprite
+        self.enemy_sprite = arcade.Sprite(
+            "Graphics/Character_animation/monsters_idle/skeleton1/v1/skeleton_v1_1.png", SPRITE_SCALING_PLAYER,
+        )
+        # Set enemy location
+        enemy_grid_x = self.x + 8
+        enemy_grid_y = self.y + 3
+        self.enemy_sprite.center_x = SPRITE_SIZE * enemy_grid_x + SPRITE_SIZE / 2
+        self.enemy_sprite.center_y = SPRITE_SIZE * enemy_grid_y + SPRITE_SIZE / 2
+
+    def draw(self):
+        if self.path:
+            arcade.draw_line_strip(self.path, arcade.color.BLUE, 2)
+
+    def on_update(self, path, end):
```

Comment:
I'd recommend changing the name of this function if it doesn't follow the "convention" of other `on_update` functions in other classes (ie, they take `deltatime` as an argument and that's it).

Changing this to another name would help make it more clear what this function does. Maybe something like, `update_path` would make sense?